src/app.ts

```diff
@@ -250,5 +250,14 @@ export function newApp(
     }
   );
 
+  app.post(
+    "/api/v1/device/register",
```

Comment:
`register` is an action, not a resource - why not just `POST /api/v1/device` ?

Reply:
we will have a register and a unregister actions